cmd/user-agent/src/router/index.js

```diff
@@ -79,7 +79,24 @@ const routes = [
                 name: "present-proof",
                 component: PresentProof,
                 meta: {requiresAuth: true}
-            }
+            },
+            {
+                path: "worker",
+                name: "chapi-worker",
+                component: WalletWorker
+            },
+            {
+                path: "StoreInWallet",
```

Comment:
These pages shouldn't be part of router children, 
We intentionally kept them out of 'router children' to hide left navigation, top navigation.

Reply:
This question is related to your other question here: https://github.com/trustbloc/edge-agent/pull/368#discussion_r494446766.

These pages need to be served by the backend in order to be reachable by the `authn.io` credential mediator domain.

Here's some logging output from the backend while storing a presentation using `/wallet/webwallet`:

![image](https://user-images.githubusercontent.com/2019896/94204874-16d49e00-fe90-11ea-9e29-9e6450fd6bbd.png)

Reply:
I don't really understand your comments about "hiding left navigation, top navigation"

Reply:
@sudeshrshetty fixed, here is how the CHAPI layout looks:

![image](https://user-images.githubusercontent.com/2019896/94210403-a2ecc280-fe9c-11ea-90af-a4b5be3708de.png)

![image](https://user-images.githubusercontent.com/2019896/94210415-aa13d080-fe9c-11ea-9231-3c9ee54e5b8b.png)

Reply:
@sudeshrshetty disabled the footer on the CHAPI windows as per your request (reused the `hideFooter` route metadata)